crates/astria-conductor/src/sequencer/snapshots/astria_conductor__sequencer__reporting__tests__snapshots.snap

```diff
@@ -3,6 +3,6 @@ source: crates/astria-conductor/src/sequencer/reporting.rs
 expression: ReportRollups(block.rollup_transactions())
 ---
 {
-  "2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a": 2,
-  "4545454545454545454545454545454545454545454545454545454545454545": 1
+  "KioqKioqKioqKioqKioqKioqKioqKioqKioqKioqKio=": 2,
```

Comment:
This snapshot is updated because the display-formatting of `RollupId` changed from hex to base64 to be compliant with protobuf <> JSON mapping (and because that's just what `pbjson` gives us)